scripts/install.sh

```diff
@@ -11,6 +11,7 @@ KEEP_CORE_PATH=$PWD
 KEEP_BEACON_SOL_PATH="$KEEP_CORE_PATH/solidity/random-beacon"
 KEEP_ECDSA_SOL_PATH="$KEEP_CORE_PATH/solidity/ecdsa"
 TMP="$KEEP_CORE_PATH/tmp"
+OPENZEPPELIN_MANIFEST=".openzeppelin/*-1101.json"
```

Comment:
Maybe we should do `unknown-*.json` https://docs.openzeppelin.com/upgrades-plugins/1.x/network-files
```suggestion
OPENZEPPELIN_MANIFEST=".openzeppelin/unknown-*.json"
```

Reply:
ok, I think we can do it. Basically, I wanted to remove the 'development' manifest, but I think we can fresh install all the local chains. [61bccab](https://github.com/keep-network/keep-core/pull/3131/commits/61bccabd8663dc2a0af3a1a2166bc0bd7741ec78)